lib/photos/actions.ts

```diff
@@ -0,0 +1,31 @@
+import { db } from '@/lib/db';
+import { photos } from '@/lib/db/schema';
+import { Photo, NewPhoto } from '@/lib/db/schema/photos';
+
+export class PhotosActions {

```

Comment:
Please do not write any classes in next.js. Use server-side actions like we do in any other file.